src/mocks/handlers.js

```diff
@@ -0,0 +1,98 @@
+import { graphql } from 'msw';
+
+export const handlers = [
+  // Handles a "GetBreeds" query
```

Comment:
Pretty redundant. Most likely will get outdated if something gets fixed down the file.